source/sass/components/@govau/_au-body.scss

```diff
@@ -55,21 +55,64 @@
     @include AU-sronly();
   }
 
-  // The standard margin top used for uikit.
+  $spacing-unit: 0.25rem;
+
+  @function au-space($value) {
```

Comment:
Perhaps we can change this function name to something else since we already have a mixin called AU-space()? Probably also should use the `au` prefix since this function is not part of the Gold design system.